packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -39,23 +40,36 @@ export default class WebpackPlugin extends PluginBase<WebpackPluginConfig> {
 
   private port = DEFAULT_PORT;
 
+  private loggerPort = DEFAULT_LOGGER_PORT;
+
   constructor(c: WebpackPluginConfig) {
     super(c);
 
     if (c.port) {
-      if (c.port < 1024) {
-        throw new Error(`Cannot specify port (${c.port}) below 1024, as they are privileged`);
-      } else if (c.port > 65535) {
-        throw new Error(`Port specified (${c.port}) is not a valid TCP port.`);
-      } else {
+      if (this.isValidPort(c.port)) {
         this.port = c.port;
       }
     }
+    if (c.LoggerPort) {
```

Comment:
These will have to be updated for the new config name as well 👍

Reply:
oops, I overlooked this